app/helpers/hosts_helper.rb

```diff
@@ -270,7 +270,7 @@ def show_templates
       content_tag(:th, _("Template Type")) + content_tag(:th) +
       templates.map do |kind, tmplt|
         content_tag :tr do
-          content_tag(:td, _("%s Template") % kind) +
+          content_tag(:td, kind + " " + _("Template")) +
```

Comment:
This will not be translated correctly. In some languages the kind would need to appear after the `Template`. Also _(kind) should be used so the kind is translated properly (see below)